src/index.ts

```diff
@@ -166,10 +236,11 @@ class OverlayControllerGlobal {
 
   focusTarget () {
     this.focusNext = 'target'
+    this.electronWindow.setIgnoreMouseEvents(true)
```

Comment:
I don't get why this line is needed on mac. It should already be handled by L#127

Reply:
Yeah, I agree with that. It doesn't seem necessary. Just created #30 to revert this if you want.